voila-gridstack/static/extension.js

```diff
@@ -11,13 +11,18 @@
 define(['jquery',
         'base/js/namespace',
         'nbextensions/voila-gridstack/gridstack',
-        'nbextensions/voila-gridstack/gridstack.jqueryUI_require',
         'nbextensions/voila-gridstack/voila-gridstack'
        ],
-       function($, Jupyter, gridstack, _, voila_gridstack) {
+       function($, Jupyter, gridstack, voila_gridstack) {
+
+    var GRIDSTACK_STYLES = 'https://cdn.jsdelivr.net/npm/gridstack@2.0.0/dist/gridstack.min.css';
+    var grid;
 
     function load_ipython_extension() {
 
+        // import gridstack styles
+        $('head').append( $('<link rel="stylesheet" type="text/css" />').attr('href', GRIDSTACK_STYLES) );
```

Comment:
Loading the stylesheet once on init should fix the issue mentioned in https://github.com/voila-dashboards/voila-gridstack/pull/56#issuecomment-690315751

For reference, the GridStack CSS is scoped to `.grid-stack`: 

https://github.com/gridstack/gridstack.js/blob/cf393169d3961abd35d6570c3e08b429a7341cfb/src/gridstack.scss#L21

Reply:
awesome!

Reply:
I marked your earlier comment as resolved.